.github/workflows/release.yml

```diff
@@ -3,24 +3,31 @@ on:
   workflow_dispatch:
     inputs:
       version:
-        description: Version to release
-        required: true
+        description: Version to release (optional)
+        required: false
       force:
         description: Force a release even when there are release-blockers (optional)
         required: false
+  schedule:
+    # We want the release to be at 9-10am Pacific Time
+    # We also want it to be 1 hour before the on-prem release
```

Comment:
I remember our `release` workflows are quite slow, is 1 hour lead time enough here?

Reply:
Same as relay, lets roll with this for now.